hydra-main/src/main/java/com/addthis/hydra/query/web/HttpQueryHandler.java

```diff
@@ -284,6 +284,32 @@ private void fastHandle(ChannelHandlerContext ctx, FullHttpRequest request, Stri
                 writer.write(q.getPaths()[0]);
                 break;
             }
+            case "/mqmaster/deactivate": {
+                log.trace("Received MeshQueryMaster deactivate request");
+                String shouldDeactivate = kv.getValue("deactivate");
+                boolean success;
+                switch (shouldDeactivate) {
+                    case "1":   success = meshQueryMaster.deactivateMqMaster(true);
+                        break;
+                    case "0":   success = meshQueryMaster.deactivateMqMaster(false);
+                        break;
+                    default:  success = false;
+                }
+                if(success) {
+                    writer.write("[1]");
+                } else {
+                    writer.write("[0]");
+                }
+                break;
+            }
+            case "/mqmaster/healthcheck": {
+                if(meshQueryMaster.getIsActive().get()) {
+                    writer.write("[1]");
+                } else {
+                    writer.write("[0]");
```

Comment:
I'm curious as to why the brackets around the numbers in the output here?

Reply:
The plan was to have the mq masters under a load balancer. The lb we received (at the time, Ops created it for us) was configured to use "[1] or [0]" as the healthcheck.

Reply:
Edited healthcheck to be Http 200/400 for now, I made a note to look at this once again when there's more clarity on what the lb will look like